pants.toml

```diff
@@ -20,7 +17,7 @@ config = "tools/pyproject.toml"
 
 [python]
 enable_resolves = true
-interpreter_constraints = [">=3.8,<3.10"]
+interpreter_constraints = [">=3.8,<3.12"]
```

Comment:
Necessary so that the tools.pex's compare equal

Reply:
Any reason why it's 3.8? Can this be bumped?